app/views/kss/home/styleguide2.html.erb

```diff
@@ -0,0 +1,742 @@
+
+        <hr>
+
+        <h2 id="card-things">
+          Card Things
+        </h2>
+
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+        <hr>
+
+        <h2 id="widgets">
+          Widgets
+        </h2>
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+        <hr>
+
+        <h2 id="avatars">
+          Avatars
+        </h2>
+
+        <div class="row">
+          <div class="col-md-4">
+            <div id="profile-avatar">
+              <img src="/images/profile-no-image.png" data-bindattr-1612="1612">
+              <div id="profile-avatar-hover">
+                <span class="button-secondary button--green fileinput-button">
+                  <span>Upload New</span>
+                  <input id="profile_avatar" class="ember-text-field" type="file" name="file" accept=".jpg,.jpeg,.tiff,.gif,.png">
+                </span>
+              </div>
+              <div class="profile-avatar-spinner"></div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <img src="/images/profile-no-image.png" data-bindattr-1041="1041" class="user-thumbnail">
+          </div>
+          <div class="col-md-4">
+            <img alt="User Name" class="user-thumbnail-small" src="/images/profile-no-image.png" data-toggle="tooltip" title="" data-original-title="User Name">
+          </div>
+        </div>
+
+        <hr>
+
+        <h2 id="task-cards">
+          Task Cards
+        </h2>
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+      </div>
+
+    </div>
+  </div>
+
+  <div class="navigation" style="left: 0; display: none;">
+    <div class="navigation-title">
+      <span class="navigation-close">×</span>Tahi
+    </div>
+    <a href="#" class="navigation-item">Colors</a>
+    <a href="#" class="navigation-item">Typography</a>
+    <a href="#" class="navigation-item">Cards</a>
+    <a href="#" class="navigation-item">System Overlay</a>
+    <a href="#" class="navigation-item">Navigation</a>
+    <a href="#" class="navigation-item">Buttons</a>
+    <a href="#" class="navigation-item">Forms</a>
+    <a href="#" class="navigation-item">Edit/Delete</a>
+    <a href="#" class="navigation-item">File Upload</a>
+    <a href="#" class="navigation-item">Contributors</a>
+    <a href="#" class="navigation-item">Error-handling</a>
+    <a href="#" class="navigation-item">Tooltips</a>
+    <a href="http://invisionapp.com/somewhere" class="navigation-item" target="_blank">Tahi Overview</a>
+    <hr>
+    <a href="#" class="navigation-item navigation-item--account">
+      <img src="/images/profile-no-image.png" class="user-thumbnail">
+      <span>
+        User Name
+      </span>
+    </a>
+    <a href="#" class="navigation-item">Sample Nav</a>
+  </div>
+
+</body>
+</html>
```

Comment:
we are going to want to move styleguide2..erb.html to something like "styleguide_template.erb", which when compiles to styleguide.erb.html via the rake task.